src/DynamoCoreWpf/ViewModels/Core/NodeViewModel.cs

```diff
@@ -660,6 +661,18 @@ public override bool IsCollapsed
                 RaisePropertyChanged(nameof(NodeWarningBarVisible));
             }
         }
+        /// <summary>
+        /// Used as a flag to indicate to associated connectors what ZIndex to be drawn at.
+        /// </summary>
+        public bool NodeInCollapsedGroup
```

Comment:
Is this meant to be serialized? @M-JULIANI @QilongTang ?

Reply:
That explain the serialization failure, please add [JsonIgnore]

Reply:
And for boolean, please use IsNodeInCollapsedGroup

Reply:
The answer if def no. That's causing it.

Reply:
As Aaron suggested, can you change the name to IsNodeInCollapsedGroup.

Reply:
Done.